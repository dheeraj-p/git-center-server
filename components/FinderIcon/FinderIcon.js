export default function FinderIcon({ type }) {
  return (
    <img src={`/images/${type}.png`} className='mr-2' width='18' height='18' />
  );
}
