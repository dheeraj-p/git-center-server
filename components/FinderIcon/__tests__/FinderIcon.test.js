import { shallow } from 'enzyme';
import FinderIcon from '../FinderIcon';

describe('FinderIcon', () => {
  it('should render properly for type blob', () => {
    const wrapper = shallow(<FinderIcon type='blob' />);
    expect(wrapper.debug()).toMatchSnapshot();
  });

  it('should render properly for type tree', () => {
    const wrapper = shallow(<FinderIcon type='blob' />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
