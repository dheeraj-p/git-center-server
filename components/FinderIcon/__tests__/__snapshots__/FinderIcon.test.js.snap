// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FinderIcon should render properly for type blob 1`] = `"<img src=\\"/images/blob.png\\" className=\\"mr-2\\" width=\\"18\\" height=\\"18\\" />"`;

exports[`FinderIcon should render properly for type tree 1`] = `"<img src=\\"/images/blob.png\\" className=\\"mr-2\\" width=\\"18\\" height=\\"18\\" />"`;
